pipelinewise/fastsync/mongodb_to_bigquery.py

```diff
@@ -31,7 +31,7 @@
 LOCK = multiprocessing.Lock()
 
 
-def tap_type_to_target_type(mongo_type):
+def tap_type_to_target_type(mongo_type, *_):
```

Comment:
why change the signature?

Reply:
same with the other files

Reply:
This makes all `tap_type_to_target_type`
functions are compatible with `assert_all_columns_exist`
by making them  all take a second parameter.

Reply:
I was seeing a bunch of failures in the tests due to the signature not matching here: https://github.com/transferwise/pipelinewise/blob/6934b3851512dbdd4280790bf253a0a13ab65684/tests/end_to_end/helpers/assertions.py#L346